pkg/controller/job/jobcontroller.go

```diff
@@ -540,7 +540,7 @@ func newCondition(conditionType batch.JobConditionType, reason, message string)
 	}
 }
 
-// getStatus returns no of succeeded and failed pods running a job
+// getStatus returns number of succeeded and failed pods in a running job
```

Comment:
How about: 
```go 
// getStatus returns the number of succeeded and failed pods from a given list of pods 
```

This function actually has nothing to do with jobs, although it's used by job controller